tests/unit/dependency/test_params.py

```diff
@@ -52,7 +53,7 @@ def test_params_error(dvc, params):
         loads_params(Stage(dvc), params)
 
 
-def test_loadd_from(dvc):
+def test_load_from(dvc):
```

Comment:
Please don't change the name as this is testing `loadd()` method.

Reply:
Oops, didn't do this on purpose, I thought I added an extra d by mistake and did not check the diff before pushing.